charts/ext-postgres-operator/values.yaml

```
@@ -98,10 +98,21 @@ volumeMounts: []
 
 # Existing secret where values to connect to Postgres are defined.
 # If not set a new secret will be created, filled with information under the postgres key above.
+# If ExternalSecret is set, existingSecret is ignored.
 existingSecret: ""
 
-# Additionnal environment variables to add to the pod (map of key / value)
-env: {}
+# Support for ExternalSecret Operator to fetch Postgres credentials from an external secret store.
+ExternalSecret: {}
```

```suggestion
externalSecret: {}
```
variables start with a lowercase